circuits/core/events.py

```diff
@@ -136,7 +136,7 @@ def __repr__(self):
 
         data = "%s %s" % (
             ", ".join(repr(arg) for arg in self.args),
-            ", ".join("%s=%s" % (k, repr(v)) for k, v in self.kwargs.items())
+            ", ".join("%s=%r" % (k, v) for k, v in self.kwargs.items())
```

Comment:
of course python3 has  %r. Why doing explicit call of repr() if string interpolation is already able to do it?